src/main/resources/assets/securitycraft/lang/en_us.json

```diff
@@ -498,17 +498,16 @@
 	"gui.securitycraft:scManual.viewActivatedBlock": "View activated: This block can be activated by hovering over it with your cursor while in block-breaking range.",
 	"gui.securitycraft:scManual.explosiveBlock": "Explosive: This block is able to be remotely detonated, and may be able to be activated/deactivated, using the Mine Remote Access Tool.",
 	"gui.securitycraft:scManual.customizableBlock": "Customizable: You may use the Universal Block Modifier and the various modules added by SecurityCraft to customize the functions of this block.",
-    "gui.securitycraft:scManual.options": "Available options:",
-    "gui.securitycraft:scManual.modules": "Available modules:",
+    	"gui.securitycraft:scManual.options": "Available options:",
+    	"gui.securitycraft:scManual.modules": "Available modules:",
 	"gui.securitycraft:scManual.recipe.keypad": "Place down a Frame and rightclick it with a Key Panel.",
 	"gui.securitycraft:scManual.recipe.keypad_chest": "Place down a Chest and rightclick it with a Key Panel.",
 	"gui.securitycraft:scManual.recipe.keypad_furnace": "Place down a Furnace and rightclick it with a Key Panel.",
 	"gui.securitycraft:scManual.recipe.reinforced": "Any reinforced block can be crafted as follows, if not otherwise specified in this book. Place down the vanilla equivalent (e.g. Reinforced Stone -> Stone) and break it with a Universal Block Reinforcer, or rightclick the Universal Block Reinforcer, put it into the slot and close the GUI.",
 	"gui.securitycraft:scManual.recipe.reinforced_iron_trapdoor": "Any reinforced block can be crafted as follows, if not otherwise specified in this book. Place down the vanilla equivalent (e.g. Reinforced Stone -> Stone) and break it with a Universal Block Reinforcer, or rightclick the Universal Block Reinforcer, put it into the slot and close the GUI.",
-	"gui.securitycraft:scManual.recipe.adminTool": "The Admin Tool is only available via the creative inventory, if it is enabled via the config file.",
 	"gui.securitycraft:scManual.recipe.admin_tool": "The Admin Tool is only available via the creative inventory, if it is enabled via the config file.",
 	"gui.securitycraft:scManual.recipe.reinforced_crystal_quartz": "Made from regular Crystal Quartz Blocks in a Universal Block Reinforcer.",
-	"gui.securitycraft:scManual.recipe.sc_manual": "The SecurityCraft Manual is only available via the creative inventory or by typing \"/sc help\" (without quotes).",
+	"gui.securitycraft:scManual.recipe.sc_manual": "The SecurityCraft Manual is available via the creative inventory, by typing \"/sc help\" (without quotes), or by crafting a book together with iron bars.",
```

Comment:
Unnecessary change because the recipe is shown on the manual page. We don't need an extra part in the text telling the user about that recipe. Please revert the line to how it was before. Apart from that, I don't think this line is used anymore.